dashmips/run.py

```diff
@@ -38,6 +42,8 @@ def next_instruction(program: MipsProgram):
     match = re.match(instruction_fn.regex, line)
     if match:
         # Instruction has the correct format
+        if match.regs[0][1] + 1 < match.endpos:
+            raise MipsException("Too many or incorrectly formatted arguments.")
```

Comment:
What does this check for exactly?
Let's add a comment saying so and move the comment above lower since its confusing to see that its correct but then we throw an error saying its incorrect

Reply:
I must have written this and the below code with an older or malformed version of the code, because somehow the regex wasn't functioning/catching those errors when I was writing my tests (which was very strange to me) so I wrote these to correct that.  My code should be updated now and I tried deleting this code to see if it functions correctly and it looks like it does!  I'll remove both this and the below from the branch.